codegenerator/cli/src/hbs_templating/codegen_templates.rs

```diff
@@ -358,14 +357,9 @@ pub struct EventTemplate {
 
 impl EventTemplate {
     pub fn from_config_event(config_event: &system_config::Event) -> Result<Self> {
-        let name = config_event
-            .get_event()
-            .name
-            .to_owned()
-            .to_capitalized_options();
+        let name = config_event.name.to_capitalized_options();
```

Comment:
I'm just trying to think if there is any place in the code where we need the orginal name of the event but I don't think so...